doc/source/development/contributing_documentation.rst

```diff
@@ -157,21 +157,3 @@ just built::
 
 And you'll have the satisfaction of seeing your new and improved documentation!
 
-.. _contributing.dev_docs:
-
-Building main branch documentation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When pull requests are merged into the Xorbits ``main`` branch, the main parts of
-the documentation are also built by readthedocs. These docs are then hosted `here
-<https://doc.xorbits.io/en/latest/>`__, see also
-the :any:`Continuous Integration <contributing.ci>` section.
```

Comment:
https://doc.xorbits.io/en/latest/development/contributing_codebase.html#testing-with-continuous-integration

Since we've removed the `readthedocs` github action from CI. I think we can also update the screen-shot in the section above.

Please also pay attention to the size of the screen-shot since it may take a lot of time to render a large picture.